### ISSUE TITLE
Can't place new order via WS2 after reconnect

### ISSUE THREAD

**Issue by dubrovsky-kiril** [bitfinexcom/bitfinex-api-node#274]
Did someone had this kind of problem? After successful reconnecting and authorization order placing does not work. I am using personal logic implementation, but problem seems like is general. Please share you thoughts, experience :)

**Comment by ddeath** [comment]
Hi, what do you mean by `Cant place new order`? Do you get an error?

It is working fine for me. I am using auto reconnect. Are you using it too?

**Comment by dubrovsky-kiril** [comment]
Yes I do. Hm... How fast do you reconnect? What time interval do you use? I don't get any error. Hm... Maybe problem is in my logic of how I use post order context. Will check this later today.

**Comment by ddeath** [comment]
I do not set any time intervals. I just use auto-reconnect feature.
